### ISSUE TITLE
Cannot read properties of undefined on latest release

### ISSUE THREAD

**Issue by jprecuch** [MeilCli/slack-upload-file#856]
[Please easy english or easy japanese]

## Description
After new release, following error happens in our pipelines
```
Run MeilCli/slack-upload-file@v3
/usr/bin/docker exec  c85ff6f7f1bc0117e3eee4598cae8d07550def6c0f4edef206e3d3e1bc7085fa sh -c "cat /etc/*release | grep ^ID"
Error: Cannot read properties of undefined (reading 'id')
```
## Environment
```
      - name: Send debug logs to a Slack thread
        if: inputs.slack-notifications && always()
        uses: MeilCli/slack-upload-file@v3
        with:
          slack_token: ${{ secrets.SLACK_APP_TOKEN }}
          channel_id: ${{ inputs.slack-notifications-channel }}
          file_path: ${{ env.DEBUG_LOGS_FILE }}
          initial_comment: ${{ github.event.action }}
          thread_ts: ${{ steps.slack-notifications.outputs.thread_ts }}
          file_type: text
          file_name: ${{ github.run_id }}
          title: ${{ github.run_id }}
```

**Comment by bluebamus** [comment]
I have same issues

**my code :**
`- name: send env file to slack channel
        uses: MeilCli/slack-upload-file@v3
        with:
          slack_token: ${{ secrets.SLACK_READ_WRITE_TOKEN }}
          channel_id: ${{ secrets.SLACK_CHANNEL_ID }}
          file_path: "./.env"
          file_name: ".env"
          file_type: "text"
          initial_comment: ".env file transfer success"`


**error result :** 
`Run MeilCli/slack-upload-file@v3
  with:
    slack_token: ***
    channel_id: ***
    file_path: ./.env
    file_name: .env
    file_type: text
    initial_comment: .env file transfer success
    file_path_follow_symbolic_links: true
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.***.6/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.***.6/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.***.6/x64
    Python[2](https://github.com/devspoon/devspoon-portfolio-blog/actions/runs/6755909268/job/18364818823#step:12:2)_ROOT_DIR: /opt/hostedtoolcache/Python/[3](https://github.com/devspoon/devspoon-portfolio-blog/actions/runs/6755909268/job/18364818823#step:12:3).***.6/x6[4](https://github.com/devspoon/devspoon-portfolio-blog/actions/runs/6755909268/job/18364818823#step:12:4)
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.***.[6](https://github.com/devspoon/devspoon-portfolio-blog/actions/runs/6755909268/job/18364818823#step:12:6)/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.***.6/x64/lib
Error: Cannot read properties of undefined (reading 'id')`

**Comment by the-Jinxist** [comment]
I have the same issue, created an issue ticket here: https://github.com/MeilCli/slack-upload-file/issues/855

**Comment by MeilCli** [comment]
Thanks reporting issue.
I just published [v3.1.0](https://github.com/MeilCli/slack-upload-file/releases/tag/v3.1.0), and it contains fixing this problem.
